x/concentrated-liquidity/math.go

```diff
@@ -90,3 +88,19 @@ func getNextSqrtPriceFromAmount0RoundingUp(sqrtPriceCurrent, liquidity, amountRe
 func getNextSqrtPriceFromAmount1RoundingDown(sqrtPriceCurrent, liquidity, amountRemaining sdk.Dec) (sqrtPriceNext sdk.Dec) {
 	return sqrtPriceCurrent.Add(amountRemaining.Quo(liquidity))
 }
+
+func getLiquidityFromAmounts(sqrtPrice, sqrtPriceA, sqrtPriceB sdk.Dec, amount0, amount1 sdk.Int) (liquidity sdk.Dec) {
```

Comment:
would you be able to add a brief comment for this? (Doesn't have to be formal since we're going to come back with the comments and documentations) Just thought it'd be useful to have so that I could use this method in the future as well if needed